packages/mdc-tabs/README.md

```diff
@@ -211,6 +211,19 @@ dots.addEventListener('click', function (evt) {
 })
 ```
 
+### Sass Mixins
+
+To customize the label ink color or the icon ink color you can use the following mixins. You would use these mixins on the `.mdc-tab:not(.mdc-tab--active)` selector or on the `.mdc-tab--active` selector to alter label or icon colors in the activated state.
+
+#### `mdc-tab-ink-color`
+Use this mixin to set the color of icon and label to have the same color. Do not use this if you are using the `mdc-tab-label-ink-color` or `mdc-tab-icon-ink-color` mixins.
```

Comment:
Change this to:

Use this mixin to set the color of all ink on the tab.

...This give us flexibility to add new "ink" components in the future.